mosaic/mosaic.py

```diff
@@ -94,6 +96,7 @@ def run(args, client=None):
         'project': args['project'],
         'begin_date': args['begin_date'],
         'end_date': args['end_date'],
+        'end_state': args['end_state'],
```

Comment:
Can you change this to args.get('end_state', 'Done')? This is an artifact of a poorly-designed/implemented path through the code: When Finishline calls mosaic it passes a dictionary of config options, and this doesn't get the mosaic argparse default options, so this would fail unless Finishline were updated to include the new end_state option in the dict.

I'm not sure how best to change the code to fix this.

Reply:
Yup, done.